### PR TITLE
ICMSLST-486 Add manage and view derogations for ILB Admin

### PR DESCRIPTION
**Pull request by kevinetienne**
__Merging to a feature branch__

Add the ability to take/release ownership of a 'Derogation' application and to view the application.

### CODE REVIEW

File: web/domains/case/_import/views.py [hunk 1]
```diff
@@ -363,7 +369,17 @@ def _view_sanctions_and_adhoc_case(request, application):
...
+    context = {
+        "process_template": "web/domains/case/import/partials/process.html",
+        "process": application,
+        "page_title": application.application_type.get_type_description(),
+        "supporting_documents": application.supporting_documents.filter(is_active=True),
+    }
+    return render(request, "web/domains/case/import/view_derogations.html", context)
+
+
+def _view_case(request: HttpRequest, application: ImportApplication) -> HttpRequest:
```
Comment (MattHolmes123) [inline web/domains/case/_import/views.py:382]: _view_case returns a HttpResponse

**Review by MattHolmes123 — CHANGES_REQUESTED** [review]
Added a comment

File: web/domains/case/_import/views.py (hunk 1 above)
Comment (kevinetienne) [reply web/domains/case/_import/views.py:382]: oh yes interesting it hasn't been picked up by mypy

Comment (kevinetienne) [reply web/domains/case/_import/views.py]: fixed in https://github.com/uktrade/icms/pull/347/commits/209e4c0803b2eae0612e768f3a7eb454cb2a5ec7

**Review by oskusalerma — APPROVED** [review]

**Review by MattHolmes123 — APPROVED** [review]

**Pull request merged**